### PR TITLE
[1/3] Applications endpoint

### PR DESCRIPTION
**Pull request by zsinx6**
This PR focus is to divide the changes into smaller pr to be easier to review

It covers only the create application (e.g. POST to /applications), and is working with this branch of jobbergate-cli:  https://github.com/omnivector-solutions/jobbergate-cli/tree/local_fastapi

The next ones will cover DELETE and PUT respectively

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ install:
 	poetry install
 
 run:
-	poetry run uvicorn jobbergateapi2.main:app --reload
+	poetry run uvicorn --host 0.0.0.0 jobbergateapi2.main:app --reload
```
Comment (zsinx6) [inline Makefile:11]: This makes the server visible in the local network

File: jobbergateapi2/apps/applications/routers.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                "application_file": application_file,
+                "application_config": application_config,
+            }
+            application_created_id = await database.execute(query=query, values=values)
+            application.id = application_created_id
+
+        except INTEGRITY_CHECK_EXCEPTIONS as e:
+            raise HTTPException(status_code=422, detail=str(e))
+    application_location = (
+        f"{settings.S3_BASE_PATH}/TEST/applications/{application.id}/jobbergate.tar.gz"
```
Comment (zsinx6) [inline jobbergateapi2/apps/applications/routers.py:57]: This is here only for testing, will be the line below once finished

**Comment by jamesbeedy** [comment]
`sqlite.db` - add this to `.gitignore`

**Comment by zsinx6** [comment]
> `sqlite.db` - add this to `.gitignore`

https://github.com/omnivector-solutions/jobbergate-api-fastapi/blob/85a6ae383e0caa5fb94b72b9290a410d589d7db4/.gitignore#L113 it is already there

**Review by jamesbeedy — APPROVED** [review]
👍

**Pull request merged**